lldb/source/Commands/CommandObjectStart.h

```diff
@@ -0,0 +1,32 @@
+//===-- CommandObjectStart.h ---------------------------------*- C++ -*-===//
```

Comment:
Revert this source file since we are just adding an option to "process launch" right?